next.config.js

```diff
@@ -0,0 +1,8 @@
+/** @type {import('next').NextConfig} */
+const nextConfig = {
+    output: "export",
+    // https://cdn.openfin.co/health/deployment/index.html
+    basePath: "/health/deployment/alpha",
```

Comment:
can it use `github.event.inputs.enviroment ` somehow here, instead of hard-coding

Reply:
I updated it - sets it dynamically noe